CHANGELOG.md

```diff
@@ -12,6 +12,17 @@ The format is based on [Keep a Changelog], and this project adheres to
 * `Fixed` for any bug fixes.
 * `Security` in case of vulnerabilities.
 
+## [UNRELEASED]
+
+### Added
+
+* Load model state dict into a new model with modified `RPUConfig`. (\#276)
+* Visualization for noise models for analog inference hardware simulation. (#278)
```

Comment:
could you change `#` --> `\#`  to make it consistent?

Reply:
Absolutely! Fixed at 7435466

Reply:
Many thanks!